src/main/java/greencity/repository/FavoritePlaceRepo.java

```diff
@@ -35,7 +35,7 @@
      * @param email   - user's email
      * @author Zakhar Skaletskyi
      */
-    void deleteByPlaceIdAndUserEmail(Long placeId, String email);
+    int deleteByPlaceIdAndUserEmail(Long placeId, String email);
```

Comment:
deleteByPlaceIdAndUserEmail() method shoud return Long

Reply:
Ids return not id